lib/project.rb

```diff
@@ -108,8 +108,6 @@ def validate_paths
 
   def normalize_paths
     @driver_path.chomp!('/')
-    @config['images_path'].chomp!('/')
```

Comment:
Don't we need something like this in virthck.rb code?

Reply:
We have something like this under /engines/virthck.rb